### PR TITLE
add proposals panel to /admin/users/:id page

### PR DESCRIPTION
**Pull request by pkarman**
small developer enhancement

### CODE REVIEW

File: app/admin/user.rb [hunk 1]
```diff
@@ -38,5 +38,14 @@
       row :updated_at
       row('Roles') { user.roles.map(&:name).join(', ') }
     end
+
+    panel "Proposals" do
+      table_for user.proposals.order('created_at DESC') do |tbl|
+        tbl.column('ID') { |proposal| link_to proposal.public_id, admin_proposal_path(proposal) }
+        tbl.column('Name') { |proposal| proposal.name }
+        tbl.column('Date') { |proposal| proposal.created_at }
```
Comment (jessieay) [inline app/admin/user.rb]: maybe "Created at` ?

**Comment by jessieay** [comment]
One comment on column name, otherwise looks good to QA!

**Comment by pkarman** [comment]
thanks for column name comment. I changed it to be the same as used in all the proposal templates: `Submitted`

**Comment by jessieay** [comment]
QAing now!

**Comment by jessieay** [comment]
having trouble pushing to cloud foundry -- will QA when I am able to push successfully

**Comment by jessieay** [comment]
now that issue pushing is fixed, I will QA this once @pkarman is done w c2-dev

**Comment by jessieay** [comment]
looks good on c2-dev, merging!

**Pull request merged**